lib/ansible/plugins/cliconf/nxos.py

```diff
@@ -99,6 +110,8 @@ def get_device_info(self):
         if match_os_platform:
             device_info['network_os_platform'] = match_os_platform.group(1)
 
+        device_info['network_os_json_support'] = self.check_json_support()
```

Comment:
Instead of checking for json support and adding `network_os_json_support ` key to indicate json support please add `output` key which is a list of support output formats to be in sync with other cliconf plugins (ios, eos, vyos)